service/create/cert_signer.go

```diff
@@ -0,0 +1,73 @@
+package create
+
+import (
+	"fmt"
+	"strings"
+
+	microerror "github.com/giantswarm/microkit/error"
+)
+
+type issueResponse struct {
+	Certificate  string
+	PrivateKey   string
+	IssuingCA    string
+	SerialNumber string
+}
+
+// Issue generates a certificate using the PKI backend signed by the certificate
+// authority associated with the configured cluster ID.
+func (s *Service) Issue(config CertificateSpec) (issueResponse, error) {
```

Comment:
@xh3b4sd yes it is the certctl code. I see now that it makes sense to reuse the library rather than copying across. I'll look into that now and redo the PR.